enhancements/installer/kubevirt-ipi.md

```diff
@@ -0,0 +1,341 @@
+---
+title: kubevirt-platform-provider
+authors:
+  - "@ravidbro"
+reviewers:
+
+approvers:
+
+creation-date: 2020-07-14
+last-updated: 2020-07-14
+status: implementable
+---
+
+# kubevirt-platform-provider
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+## Summary
+
+This document describes how `kubevirt` becomes a platform provider for Openshift. \
```

Comment:
What are the trailing slashes?

also, s/Openshift/OpenShift/

Reply:
The trailing backslashes are for new line in the .md file

Reply:
Fixed